docs/reference/airnode/next/chain-idiosyncrasies.md

```diff
@@ -57,7 +59,9 @@ gas limit automatically.-->
 ### Metis
 
 On the Metis testnet Stardust, though not on the Metis mainnet Andromeda, it is
-recommended to use a `fulfillmentGasLimit` of at least `5000000`.
+recommended to use a `fulfillmentGasLimit` of at least `5000000`. If you prefer
+not to specify it, Airnode will attempt to estimate the appropriate gas limit
+automatically.
```

Comment:
nit: to me 'prefer' sounds slightly unusual here, perhaps instead something similar to the above?

If `fulfillmentGasLimit` is not specified, Airnode will attempt to estimate the appropriate gas limit
automatically.